account-kit/react/src/components/dialog/dialog.tsx

```diff
@@ -40,19 +69,24 @@ export const Dialog = ({ isOpen, onClose, children }: DialogProps) => {
     setScrollLocked(getComputedStyle(document.body).overflow !== "hidden");
   }, []);
 
-  return isOpen
+  return renderPortal
     ? createPortal(
         <RemoveScroll enabled={isScrollLocked}>
           {/* Overlay */}
           <div
             aria-modal
             role="dialog"
-            className="fixed inset-0 bg-black/80 flex items-end md:items-center justify-center z-[999999] animate-fade-in"
+            ref={backdropRef}
```

Comment:
I think we want to listen to the animation ending on the `slide-down` below not the backdrop